### DISCUSSION TITLE
commandTypes

### DISCUSSION THREAD

**Discussion by iptvcld** [dgreif/ring#1039] [General]
How can we run one of the commandTypes from a device?
'update-node-neighbors.start': { requiresTrust: false },

**Comment by iptvcld** [comment]
I tired this but did not work.
ZWaveExtender.sendCommand(update-node-neighbors.start)

**Reply by iptvcld** [reply]
@dgreif Would you know how we can send commands using for example the above commandType?

**Reply by dgreif** [reply]
I'm not familiar with the particular command you are trying to send. For the alarm system, `sendCommand` is working as expected. It looks like you were missing quotes on your command, so that _might_ be it?

`ZWaveExtender.sendCommand('update-node-neighbors.start')`
